lib/workers/branch/index.ts

```diff
@@ -391,6 +391,14 @@ export async function processBranch(
         });
       }
     }
+    if (config.rebaseWhen === 'never') {
+      logger.debug(`Skipping commit (rebaseWhen=never)`);
+      return {
+        branchExists,
+        prNo: branchPr?.number,
+        result: BranchResult.NoWork,
+      };
+    }
     config.forceCommit =
```

Comment:
Shouldn't this check be done *before* the rebaseWhen=never return? i.e.to allow manually requested rebase?

Reply:
Now, `isConflicted` won't cause rebase while dashboard or PR checkboxes should